cybox/utils/__init__.py

```diff
@@ -73,6 +74,17 @@ def _import_submodules(pkg):
         importlib.import_module(mod_name)
 
 
+def is_sequence(item):
```

Comment:
Is there a reason we want to support tuples? Why not just check if `value` is a list?